app/controllers/hubrise_app/callback_controller.rb

```diff
@@ -19,5 +20,12 @@ def ensure_app_instance_found!
     def event_params
       params.require(:callback).permit!.to_h
     end
+
+    def verify_event!
+      unless hubrise_gateway.valid_hmac?(request.raw_post,
+                                         request.headers["X-Hubrise-Hmac-Sha256"])
+        head(401)
```

Comment:
Hi @nsave can we send a response indicating the expected vs received HMac in case of verification failure? There is no reason this will ever be useful, but that could also save us some hair if we ever break something

Reply:
I don't think this is safe

Reply:
Not sure why it wouldn't be safe, but a generic "Wrong X-Hubrise-Hmac-Sha256 header received." would work for me too.

Reply:
updated

Reply:
(just understood why it was not safe, good thinking @nsave !)